### DISCUSSION TITLE
Getting raw results from Repository

### DISCUSSION THREAD

**Discussion by klausXR** [mikro-orm/mikro-orm#2612] [Q&A] [answered]
Is there a way to skip the additional work that MikroORM does when querying the database, like initializing entities?
For read queries, I'd like a way to get the objects that Knex returns, without MikroORM interventions ( or a minimal amount of it ). Is there a flag to do that?

**Comment by B4nan** [comment, marked as answer]
You can use QB, its `execute()` method will give you that. You can also use its `mapResults` argument to disable translation of column names to property names.
